dts/bindings/pwm/maxim,max31790.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 SILA Embedded Solutions GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: Maxim MAX31790 6-channel I2C-bus PWM controller
+
+compatible: "maxim,max31790"
+
+include: [pwm-controller.yaml, i2c-device.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  channel-count:
+    type: int
+    required: true
+    const: 6
+    description: number of supported channels
```

Comment:
Why is this a property if it is constant?

Reply:
I thought about handling this the same way as for instance the number of available GPIOs for a port expander, for instance in https://github.com/zephyrproject-rtos/zephyr/blob/2579690be7ecf6d19bea9df74a39ecb4c924bfc8/dts/bindings/gpio/rohm,bd8lb600fs.yaml.

Reply:
done